app/components/forms/wizard/basic-details-step.js

```diff
@@ -116,10 +116,10 @@ export default Component.extend(FormMixin, EventWizardMixin, {
     this.oldTimezone = this.data.event.timezone;
     if (!oldTimezone || !this.oldTimezone || oldTimezone === this.oldTimezone) {return}
     if (event.startsAt) {
-      event.startsAt = moment.tz(event.startsAt.clone().tz(oldTimezone).format('YYYY-MM-DDTHH:mm:ss.SSS'), moment.ISO_8601, this.data.event.timezone);
+      event.startsAt = moment.tz(event.startsAt.clone().tz(oldTimezone).format('DD MMMM, YYYY hh:mm A'), moment.ISO_8601, this.data.event.timezone);
```

Comment:
This is not user facing format. This is ISO8601 format and it needs to remain same